t/Hydra/Controller/Root/status.t

```diff
@@ -50,6 +50,12 @@ subtest "/queue" => sub {
         print STDERR Dumper $response->content;
     }
 };
+
+subtest "/search" => sub {
+    my $response = request(GET '/search?query=one');
+    ok($response->is_success, "The search page 200's.");
+};
```

Comment:
This is insufficient. Needs test coverage on matching:

- project names
- jobset names
- job names
- builds' output path
- derivation path